### ISSUE TITLE
Can `HumanDurationData` derive `Copy` ?

### ISSUE THREAD

**Issue by IgorLaborieV8** [rsalmei/human-repr#8]
I want to use the newtype pattern with `HumanDurationData`.
If I can derive `Copy`, my API will be more developper friendly.

**Comment by rsalmei** [comment]
Yes, it can indeed and actually is on the still unpublished version. I'm working on #7, and I did make it Copy there.
Thank you for the PR, but it was already done. I'll try to release this new version ASAP.

**Comment by rsalmei** [comment]
In fact, @IgorLaborieV8, the new version should have all the structs Copy!
Now I am interning units, i.e. leaking them so they'll be allocated only once, on both Count and Throughput, so all they store is a `&'static str`, which is Copy 🙂.
